cluster_management/src/main/java/com/pinterest/rocksplicator/Participant.java

```diff
@@ -60,7 +60,7 @@
   private static final String configPostUrl = "configPostUrl";
   private static final String disableSpectator = "disableSpectator";
 
-  private HelixManager helixManager;
+  private static HelixManager helixManager;
```

Comment:
Now we initialize a static data member in the constructor which is not ideal. I am not sure if there is a way to not allow the constructor gets called twice. One thing we may do is to make the Participant() constructor private. So we won't initialize helixManager more than once if we don't call main() twice.

Reply:
i can also check for isConnected() in the constructor if you think that might help as well, and only initialize if isConnected() is false